scribeauth/scribeauth.py

```diff
@@ -212,19 +201,17 @@ def get_tokens(self, **param: Unpack[RefreshToken]) -> Tokens | Challenge:
     def get_tokens(self, **param) -> Tokens | Challenge:
         """A user gets their tokens (refresh_token, access_token and id_token).
 
-        Args
-        ----
-        username -- Username (usually an email address).
+        It is possible to pass a UsernamePassword or a RefreshToken:
 
-        password -- Password associated with this username.
+        :param UsernamePassword: username (usually an email address) and password (associated with this username).
```

Comment:
:param username: username (usually an email address)
:param password: password (associated with this username)

Or 

:param refresh_token: whatev